iceoryx_hoofs/test/moduletests/test_allocator.cpp

```diff
@@ -76,7 +78,11 @@ TEST_F(Allocator_Test, allocateTooMuchSingleElement)
     ::testing::Test::RecordProperty("TEST_ID", "9deed5c0-19d8-4469-a5c3-f185d4d881f1");
     iox::posix::Allocator sut(memory, memorySize);
     std::set_terminate([]() { std::cout << "", std::abort(); });
+    // todo #1196 remove EXPECT_DEATH
+    // NOLINTBEGIN(hicpp-avoid-goto, cppcoreguidelines-avoid-goto, cert-err33-c, cppcoreguidelines-pro-type-vararg,
+    // hiccpp-vararg)
     EXPECT_DEATH({ sut.allocate(memorySize + 1, MEMORY_ALIGNMENT); }, ".*");
+    // NOLINTEND
```

Comment:
Please add the warnings from `NOLINTBEGIN` in here.

Reply:
Or you just use `NOLINTNEXTLINE` and remove `NOLINTEND`

Reply:
This is awful from the amount of text and repetitive as well. It really disturbs the flow when I am reading code. Need probably just some plugin to hide all these suppressions...